android/app/build.gradle

```diff
@@ -162,7 +162,7 @@ android {
             def abi = output.getFilter(OutputFile.ABI)
             if (abi != null) {  // null for the universal-debug, universal-release variants
                 output.versionCodeOverride =
-                        versionCodes.get(abi) * 1048576 + defaultConfig.versionCode
+                defaultConfig.versionCode * 1000 + versionCodes.get(abi)
```

Comment:
What was 1048576? Can we have a comment explaining what this maths is doing?

Reply:
Sure, I will dig it out but this was just a recommendation that came from the react native upgrade guide

Reply:
https://react-native-community.github.io/upgrade-helper/?from=0.61.5&to=0.64.0

This is just something and config from react native that they require I don't know the 100% maths they came up with but it looks like there is a comment above explaining what this section and function is

https://github.com/Skyscanner/backpack-react-native/pull/715/files/05d85662efa8e35f3a589bd8f0713d344ba71f72#diff-9526ccfd1d1813ed49c39f8c54dbeb512607376a007d824b905bc8b4e4d202d9R159

Reply:
![Screenshot 2021-08-02 at 12 53 08](https://user-images.githubusercontent.com/8831547/127857796-ed85be81-3364-461a-9423-1943a204bc45.png)